webrender/Cargo.toml

```diff
@@ -31,7 +31,7 @@ webrender_traits = {path = "../webrender_traits", default-features = false}
 bitflags = "0.7"
 
 [target.'cfg(any(target_os = "android", all(unix, not(target_os = "macos"))))'.dependencies]
-freetype = {version = "0.1.2", default-features = false}
+freetype = { git = "https://github.com/servo/rust-freetype", branch = "bindgen", default-features = false }
```

Comment:
I think we should be publishing 0.2 and using it here instead of the branch, which is about to be merged into master anyway.